src/Message/PaymentIntents/AuthorizeRequest.php

```diff
@@ -0,0 +1,374 @@
+<?php
+
+/**
+ * Stripe Payment Intents Authorize Request.
+ */
+namespace Omnipay\Stripe\Message\PaymentIntents;
+
+use Money\Formatter\DecimalMoneyFormatter;
+
+/**
+ * Stripe Payment Intents Authorize Request.
+ *
+ * An authorize request is similar to a purchase request but the
+ * charge issues an authorization (or pre-authorization), and no money
+ * is transferred.  The transaction will need to be captured later
+ * in order to effect payment. Uncaptured charges expire in 7 days.
+ *
+ * A payment method is required. It can be set using the `paymentMethod`, `source`,
+ * `cardReference` or `token` parameters.
+ *
+ * *Important*: Please note, that this gateway is a hybrid between credit card and
+ * off-site gateway. It acts as a normal credit card gateway, unless the payment method
+ * requires 3DS authentication, in which case it also performs a redirect to an
+ * off-site authentication form.
+ *
+ * Example:
+ *
+ * <code>
+ *   // Create a gateway for the Stripe Gateway
+ *   // (routes to GatewayFactory::create)
+ *   $gateway = Omnipay::create('Stripe\PaymentIntents');
+ *
+ *   // Initialise the gateway
+ *   $gateway->initialize(array(
+ *       'apiKey' => 'MyApiKey',
+ *   ));
+ *
+ *   // Create a payment method using a credit card object.
+ *   // This card can be used for testing.
+ *   $card = new CreditCard(array(
+ *               'firstName'    => 'Example',
+ *               'lastName'     => 'Customer',
+ *               'number'       => '4242424242424242',
+ *               'expiryMonth'  => '01',
+ *               'expiryYear'   => '2020',
+ *               'cvv'          => '123',
+ *               'email'                 => 'customer@example.com',
+ *               'billingAddress1'       => '1 Scrubby Creek Road',
+ *               'billingCountry'        => 'AU',
+ *               'billingCity'           => 'Scrubby Creek',
+ *               'billingPostcode'       => '4999',
+ *               'billingState'          => 'QLD',
+ *   ));
+ *
+ *   $paymentMethod = $gateway->createCard($card)->send()->getCardReference();
```

Comment:
$card has to be an array. In this example $card is an object.

Reply:
Resolved with #134